ui/src/app/layout/dashboard/dashboard.component.html

```diff
@@ -122,8 +122,10 @@ <h2 class="text-muted">Dashboard</h2>
                     <ul class="chat" *ngIf="(changelog | async)?.length">
                         <li class="left clearfix" *ngFor="let change of changelog | async">
                             <span class="chat-img float-left">
-                                <!-- TODO gravatar? MS Office 365? -->
-                                <img src="/assets/images/default_avatar.png" class="img-circle">
+                                <div class="imageContainer">
+                                      <img class="image1" src='{{getDefaultPicture(change.principal)}}'>
```

Comment:
Should be an Observable, no direct method invocation (Angular will call that method a dozen times)